Clear Project Guidelines

Computer8543 opened the pull request:
I separated this workflow from the rest and reverted some changes.

Computer8543 committed:
Initial draft of Negotiation Mastery Program in a YAML file format.

Computer8543 committed:
Merge pull request #1 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Merge Initial Draft code from main branch to begin negotiation. Also removed comments to make change so I can git commit directly to this issue.

Computer8543 committed:
Merge pull request #2 from Computer8543/main

Merge main branch changes into kyle branch

Computer8543 committed:
Merge branch 'mindset-dojo:main' into kyle

Computer8543 committed:
Create LICENSE.md

Add CC BY-SA 4.0 license to repo.

Computer8543 committed:
Create TODO.md

Create an initial draft of a minimal set of contribution guidelines for the Mindset Dojo repo.

Computer8543 committed:
Create README.md

Created README.md file to summarize the offerings of Mindset Dojo and it's community succinctly.

Computer8543 committed:
Merge pull request #3 from Computer8543/kyle

Merge kyle branch into main branch

Computer8543 committed:
Rename TODO.md to CONTRIBUTING.md

This is to follow open source naming conventions.

Computer8543 committed:
Merge pull request #4 from Computer8543/kyle

Rename TODO.md to CONTRIBUTING.md

Computer8543 committed:
Green Belt Push Request and Tag Change

Computer8543 committed:
Merge pull request #5 from Computer8543/kyle

Merge kyle branch into main

Computer8543 committed:
Delete _data/negotiation-mastery-program.yaml

Computer8543 committed:
Update kyle-ingersoll.yaml

Reverted my YAML file back to Yellow Belt status and old tag.

Computer8543 committed:
Delete CONTRIBUTING.md

Computer8543 committed:
Update README.md

I reduced it down to the three main points you requested. We can refine this further later.

michael-basil commented on README.md:
```diff
@@ -0,0 +1,7 @@
+
+See the community at https://dojo.center/community
+
+Please respect the CC BY-SA 4.0 license when contributing.
```
DRY - don't repeat yourself ... link directly to the license (in case it changes in the future)

Please respect the [LICENSE](./LICENSE.md)

michael-basil commented on README.md:
```diff
@@ -0,0 +1,7 @@
+# 🥋 Mindset Dojo & Negotiation Mastery Program
+
+Start your journey today: build emotional wisdom, master authentic conversation, and unlock your full leadership potential at https://dojo.center/
```
I would make this more mysterious .. 

Start your exploration at https://dojo.center

Computer8543 committed:
Update kyle-ingersoll.yaml

Changed Base 64 String to Central Repository String to resolve diff conflict in file.
